schedule/models/events.py

```diff
@@ -43,6 +44,8 @@ class Event(models.Model):
     calendar = models.ForeignKey(Calendar, null=True, blank=True, verbose_name=_("calendar"))
     objects = EventManager()
 
+    recent_start = models.TextField(null=True)
```

Comment:
I think you mean "If you don't need to represent an object ..." since in that case it should be easier to represent this as a json string. If we agree that Rule never changes, I can transition to json and save string representation of all the datetime fields that comprise recent_start.